### DISCUSSION TITLE
[Fast Data] avoid SV update when no actual changes happens

### DISCUSSION THREAD

**Discussion by Noste92** [mia-platform/community#484] [Q&A] [answered]
### What product are you having troubles with?

Fast Data

### What Console version are you using?

Old RTU Library

### Description

Hello,

I have a question related to the ingestion step of Fast Data. I have tested this behavior with an old version of the Real Time Updater libraries and I like to have some more information about it.

When a new message comes to Kafka related to a specific a Projection it triggers the Fast Data flow to update a the Projection on MongoDB and the related Single View.
 but if the Projection doesn't really update (I mean all the fields are the same) I think the update is triggered as well.
Do you think is there a way to avoid these updates? Or another way to approach this problem?

**Comment by danibix95** [comment, marked as answer]
Hi, could you please provide us the version of Real-Time Updater service your're referring to? Why do you consider it as a library?

Furthermore, could you please expand on the business case that requires to introduce this change in the product?

**Note:** currently both Real-Time Updater and Projection Storer services process all the CDC change events in the same manner, independently from the fact that a record has the same values of the one that were previously saved. In theory, Change Data Capture systems send change events only when a record is inserted, modified or deleted.

**Reply by Noste92** [reply]
Hi @danibix95, sorry for the delay of my reply. Your note seems clear to me, I'd only like to know the general behavior of Fast Data and what I can do today with the product. In our case we're trying to optimize the CDC to send less events.
Thank You
